packages/query/src/stacks/stacks-client.ts

```diff
@@ -71,7 +72,10 @@ export function stacksClient(basePath: string) {
       );
       return resp.data;
     },
+
     // TODO: Need to replace, this endpoint has been deprecated
+    /** @deprecated use getEventsForAddressTransaction
```

Comment:
I started to look into removing this and added a comment here: https://github.com/leather-io/mono/pull/542#discussion_r1820903384 

`getAccountTransactionsWithTransfers` uses [this endpoint](https://docs.hiro.so/stacks/api/accounts/transactions-with-transfers) which seems deprecated but no direct alternative is offered.